bananas_api/helpers/regions.py

```diff
@@ -0,0 +1,148 @@
+import json
+import os
+import unicodedata
+
+REGIONS = {}
+
+folder = os.getcwd()
+if not os.path.exists(f"{folder}/region-un-m49.csv"):
```

Comment:
This looks fragile to me, considering that PYTHONPATH is a list of paths.
I would have used
```python
folder = os.path.join(os.path.dirname(__file__), "..", "..")
```

Reply:
Neither `PYTHONPATH` nor your suggestion is really any good. Your suggestion for example breaks if you `pip install -e .` the package.

As I mentioned in the other PR, I also don't like this file is copied from one repository to the other.

In result, most likely next weekend, I will spend some time moving this code away to its own library .. either in https://github.com/OpenTTD/py-helpers or its own (py-regions?). There I will spend the effort to actually proper package such files in a pip package, so it can be properly installed. That way we don't have to deal with this stuff, and that way we are sure both repositories use the same code. Sadly, packaging non-Python files in a pip package is not trivial, so I need to take some time to make that happen :)

Long story short, if you don't mind just going for "that-what-works" for now, I promise to make it pretty soon :)